Source/Cogworks.Umbraco.Essentials/Extensions/ImageCropperExtensions.cs

```diff
@@ -8,9 +8,28 @@ namespace Cogworks.Umbraco.Essentials.Extensions
 {
     public static class ImageCropperExtensions
     {
-        public static string GetCropUrls(this IPublishedContent image, string cropAlias, int? width = null, int? height = null, bool includeRetina = true, bool enableWebP = false, int? quality = null)
+        public static string GetCropUrls(this IPublishedContent image, string cropAlias = null, int? width = null, int? height = null, bool includeRetina = true, bool enableWebP = false, int? quality = null, bool returnOnlyRetina = false)
         {
-            var imageUrl = image.GetCropUrl($"{cropAlias}");
+            var imageUrl = GetCropUrlStandard(image, cropAlias, width, height, enableWebP, quality);
+
+            if (!includeRetina)
+            {
+                return imageUrl;
+            }
+
+            var retinaUrl = GetCropUrlRetina(image, cropAlias, width, height, enableWebP, quality);
+
+            if (retinaUrl.HasValue())
+            {
+                return returnOnlyRetina ? retinaUrl : $"{imageUrl} 1x, {retinaUrl} 2x";
```

Comment:
Move `?` and `:` to new line.
```
return returnOnlyRetina
          ? retinaUrl
           : $"{imageUrl} 1x, {retinaUrl} 2x";
```

Reply:
@aochmann this is down to preferences, IMO the logic is the same